src/main/resources/licenses/custom-settings.xml

```diff
@@ -0,0 +1,35 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+                      http://maven.apache.org/xsd/settings-1.0.0.xsd">
+
+   <profiles>
+      <profile>
+         <id>local-m2</id>
```

Comment:
Do we need an equivalent version of this file for generating licenses.xml for product?

Can that be easily done in a single branch?

Reply:
Good point. I think we can use a property for jboss repo url and add one more profile with product value.

Reply:
@kenfinnigan Is it safe to share the product repo URL publicly?

Reply:
We have it present in the pom-redhat.xml so I imagine we're ok

Reply:
Ok. I forgot about the `pom-redhat.xml` - we should copy the new profile there as well.